src/Toast/index.jsx

```diff
@@ -7,21 +8,20 @@ import { Button, IconButton, Icon } from '..';
 import { Close } from '../../icons';
 
 function Toast({
-  action, children, closeLabel, delay, onClose, show,
```

Comment:
We shouldn't remove the `show` prop as it is required via `PropTypes.bool.isRequired` below.